spec/end_to_end_tests/resources/data.json

```diff
@@ -0,0 +1,30 @@
+[
+  {
+    "name": "Babel",
+    "ring": "adopt",
+    "quadrant": "tools",
+    "isNew": true,
```

Comment:
When using this JSON file, the Radar is not getting generated as the `isNew` value is not a string but a boolean. This needs to be enclosed as a string or proper validation must be added in `inputSanitizer.js` line 25 for the same. Since in CSV we are following the isNew values in all Caps, Enclosing them in a string "TRUE" , "FALSE" can resolve this issue